src/Protocol/ActivityPub/Processor.php

```diff
@@ -52,18 +52,14 @@
 class Processor
 {
 	/**
-	 * Converts mentions from Pleroma into the Friendica format
+	 * Extracts the tag character (#, @, !) from links
 	 *
 	 * @param string $body
-	 *
-	 * @return string converted body
+	 * @return string
 	 */
-	private static function convertMentions($body)
+	protected static function convertPleromaMentions(string $body): string
```

Comment:
I still don't really like the "Pleroma" in the function name. The function converts some kind of mention style, but is not said that there aren't other implementations out there who are using it as well. So it isn't exclusive for Pleroma. And yes, I wrote that "Pleroma" in the description, but I'm not sure if that function still is doing the job only for Pleroma mentions.

Reply:
The alternative that would really describe what the method does is `extractHashCharacterFromMentionLinks` but I'm not sure it would pass your conciseness test. Pleroma uses this style so we can use their name without implying the method is only relevant to Pleroma. It just converts Pleroma-style mentions.

Reply:
It could be named `normalizeMentions` since it converts mentions into a format that later can be processed by the system.

Reply:
Fair enough

Reply:
Done.